src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -39,31 +40,23 @@ public List<Publication> List()
         // TODO it maybe necessary to add authorisation to this method
         public List<PublicationViewModel> GetByTopicAndUser(TopicId topicId, UserId userId)
         {
-            var config = new MapperConfiguration(cfg =>
-            {
-                cfg.CreateMap<Publication, PublicationViewModel>();
-                cfg.CreateMap<Release, ReleaseViewModel>()
-                    .ForMember(
-                        dest => dest.LatestRelease, m => m.MapFrom(release => IsLatestRelease(release)));
-                cfg.CreateMap<Methodology, MethodologyViewModel>();
-            });
-
-            var mapper = config.CreateMapper();
-
             var publications = _context.Publications.Where(p => p.TopicId == topicId)
                 .Include(p => p.Contact)
                 .Include(p => p.Releases)
                 .Include(p => p.Methodologies)
                 .ToList();
-            
-            return mapper.Map<List<PublicationViewModel>>(publications);
+
+            return PublicationToPublicationViewModelMapper.Map<List<PublicationViewModel>>(publications);
         }
         
-        private bool IsLatestRelease(Release release)
+        private static readonly IMapper PublicationToPublicationViewModelMapper = new MapperConfiguration(cfg =>
```

Comment:
Suggest we move this out into its own mapper class

Reply:
Done